src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="search-box" 
+      data-qa="big" 
+      action="\action_page.php" 
+       method="post"
+    >
+      <button
+        class="search-btn btn--one"
+      >
+        <img 
+          class="search-img" 
+          src="images/Search.svg" 
+          alt="search_button_image"
```

Comment:
Everything that lies in this attribute will be shown to the user in the event that the image does not load. It is unlikely that such a record will be understood by users
```suggestion
          alt="Magnifying glass"
```